### DISCUSSION TITLE
Data source/model used for the API data?

### DISCUSSION THREAD

**Discussion by drmrbrewer** [weather-gov/api#452] [General]
Where is data returned from this API being pulled?  e.g. GFS?  or HRRR?  It would be really useful to know more about the underlying data.  Thanks.

**Comment by jah1007** [comment]
If you are speaking of the forecasted data like that on the /gridpoints endpoint, this data is generated by trained NWS meteorologists. While they consult the model data you list, and more, ultimately they form the forecast themselves and the result is distributed for use.

**Comment by drmrbrewer** [comment]
@jah1007 yes I'm speaking of the hourly forecast data e.g. at: https://api.weather.gov/gridpoints/TOP/31,80/forecast/hourly.  So, a centre/region (with their own NWS meteorologists) might be responsible for a grid point (or group of grid points), and they are all free to do their own thing and use their own sources to produce the forecast data (at least within constraints)?

**Reply by jah1007** [reply]
So in your case TOP is reference to the Weather Forecast Office (WFO) in Topeka, KS. These WFOs are assigned what is known as a County Warning Area (CWA) and that office is responsible for weather forecast and warning operations for that area. I cannot speak to the parameters of their training, only that I have seen multiple forecast verifications that place the Weather Service meteorologists at or near the top in weather forecasting.

**Reply by drmrbrewer** [reply]
Thanks.  And is the data for all locations (all lat/long coordinates) within an entire CWA going to be the same?  Roughly how big a geographical area is that?

**Reply by scadergit** [reply]
No, an application called AWIPS helps translate the information gathered by the meteorologist into data for each grid. The PHI office made a tour of this process: https://www.weather.gov/phi/TourAWIPS
